src/content/en/fundamentals/engage-and-retain/push-notifications/asking-permission.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: shared/narrow
+title: "Asking permission for notifications"
+description: "There's more to getting a user's permission to send them notifications than throwing up a dialog box."
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 2
+authors:
+  - josephmedley
+translation_priority: 1
+---
+
+<p class="intro">
+There's more to getting a user's permission to send them notifications than throwing up a dialog box. 
+</p>
+
+{% include shared/toc.liquid %}
+
+## Prefer logged in users
+
+If you don't know who your users are it's hard to send them relevant notifications. And if notifications aren't relevant, users might regard them as spam.
+
+
+## Trigger from a specific action
+
+Ask for permission to send notifications in response to a specific, contextual user interaction. This allows you to tie your notifications to a user's goal and makes it clear to the user why you want to send notifications. 
+
+For example, If an airline site wanted to notify users of flight delays, they would prominently display an opt-in checkbox and only ask for notification permissions after the user clicks it.
+
+<div class="mdl-grid">
+  <div class="mdl-cell mdl-cell--6-col">
+    <img src="images/airline-prompt.png" alt="Prominently display an opt-in checkbox">
+  </div>
+  <div class="mdl-cell mdl-cell--6-col">
+    <img src="images/airline-permissions.png" alt="Prominently display an opt-in checkbox"> 
+  </div>
+</div>
+
+## Clarify page-load permission requests
+
+Generally we don't believe in requesting notification permissions during the first page load. They can be distracting, spammy, and unclear. If your app requests notifications during onload, make it clear why you're asking. Dim out the screen and add an overlay that explains the reason for the request.
+
+For example:
+
+<div class="mdl-grid">
+<div class="mdl-cell mdl-cell--6-col">
```

Comment:
Sorry about that. It seems that my editor likes to auto "fix" things for me, but doesn't do so correctly.